spec/features/single_content_item_spec.rb

```diff
@@ -234,20 +234,16 @@
         expect(page).to have_selector('.govuk-link', text: I18n.t("metrics.searches.external_link"))
       end
 
-      it 'renders the metric timeseries for satisfaction' do
-        satisfaction_rows = extract_table_content(".chart.satisfaction table")
-        expect(satisfaction_rows).to match_array([
-          expected_table_dates,
-          ["User satisfaction score", "100%", "90%", "80%"]
-        ])
+      it 'renders the satisfaction score' do
+        expect(page).to have_selector('.metric-summary__satisfaction', text: '')
       end
 
       it 'renders the metric timeseries for ' do
         feedback_comment_rows = extract_table_content(".chart.feedex table")
 
         expect(feedback_comment_rows).to match_array([
-          expected_table_dates,
-          ["Number of feedback comments", "20", "21", "22"]
+        expected_table_dates,
+        ["Number of feedback comments", "20", "21", "22"]
```

Comment:
I think the indentation was a little bit better before, was this an intentional change?

Reply:
It wasn't deliberate - I think my editor did it for me... 😖